src/runtime/config/configuration-qemu.toml.in

```diff
@@ -40,6 +40,28 @@ rootfs_type=@DEFROOTFSTYPE@
 # Default false
 # confidential_guest = true
 
+# Enable pre-attestation for AMD SEV and SEV-ES guests.
+# Applies only if confidential_guest is true.
+# (default: false)
+#guest_pre_attestation = true
+#
+# A remote key broker server (KBS) is required to validate the attestation
+# measurement and inject a secret key.  
+# See https://github.com/confidential-containers/simple-kbs
+#guest_pre_attestation_kbs_uri = "@DEFGUESTPREATTESTATIONPROXY@"
+#
+# The simple-kbs can be run in "online" and "offline" modes
+#guest_pre_attestation_kbs_mode="@DEFGUESTPREATTESTATIONMODE@" 
+#
+# Keyset ID of the injected secret (offline kbs)
+#guest_pre_attestation_keyset = "@DEFGUESTPREATTESTATIONKEYSET@"
```

Comment:
I think we can maybe remove this from the config file. No need to potentially confused people with the offline stuff.